cmd/ctr/commands/run/run_windows.go

```diff
@@ -114,7 +110,7 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 			opts = append(opts, oci.WithProcessCwd(cwd))
 		}
 		if user := context.String("user"); user != "" {
-			opts = append(opts, oci.WithUsername(user))
+			opts = append(opts, oci.WithUser(user), oci.WithAdditionalGIDs(user))
```

Comment:
Even though `WithAdditionalGIDs` exits if the Windows bit of the runtime spec is filled in so it's mostly harmless, I don't think it makes sense to pass it here. It's not supported for LCOW either from reading the comment inside

Reply:
`WithUser` ends up forwarding to `WithUsername` under the hood if someone just passed a username like string so that's fine

Reply:
> I don't think it makes sense to pass it here.

You are right, I removed it.